gull_tracking/gull_tracking.ipynb

```diff
@@ -37,13 +31,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "hv.extension('bokeh')\n",
-    "\n",
-    "map_tiles  = EsriImagery().opts(alpha=1.0, width=800, height=800, bgcolor='black')\n",
-    "points     = hv.Points(df, ['longitude', 'latitude'])\n",
-    "rasterized = shade(rasterize(points, x_sampling=1, y_sampling=1, width=800, height=800), cmap=fire)\n",
+    "df = pd.read_csv('./data/HG_OOSTENDE-gps-2018.csv', usecols=['location-long', 'location-lat'])\n",
+    "df.columns = ['longitude', 'latitude']\n",
+    "df.loc[:,'longitude'], df.loc[:,'latitude'] = en_ll(df.longitude, df.latitude)\n",
     "\n",
-    "map_tiles * rasterized"
+    "df.hvplot.points(x='longitude', y='latitude', rasterize=True, tiles='EsriImagery',\n",
+    "                 cmap='fire', cnorm='eq_hist', width=800, height=800)"
```

Comment:
I think the call should include `dynspread=True` so that dots are visible when you zoom in.